tests/e2e/lib.sh

```diff
@@ -747,6 +758,17 @@ db_backup_and_restore_test() {
         fi
     fi
 
+    wait_for_api
```

Comment:
nit:  timing could be a source of flakes with this test.  If for some reason the restore is slow we COULD perform this check before the restore actually finished.  The call to `wait_for_api` has a sleep as first thing if I recall, so that helps.

Yet another reason why we need a method to put central to sleep when doing restores.